google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -509,6 +520,63 @@ public GapicSpannerRpc(final SpannerOptions options) {
     }
   }
 
+  private static String parseGrpcGcpApiConfig() {
+    InputStream inputStream = GapicSpannerRpc.class.getResourceAsStream(API_FILE);
```

Comment:
I think you can simplify this method somewhat by using (something like) `com.google.common.io.Resources.toString(resourceClass.getResource(filename), Charset.forName("UTF8"));`

Reply:
Thanks! Done.